jetson/percep/meson.build

```diff
@@ -51,6 +67,7 @@ obs_record = get_option('obs_record')
 perception_debug = get_option('perception_debug')
 write_frame = get_option('write_frame')
 data_folder = get_option('data_folder')
+vm = get_option('vm')
```

Comment:
why are variables from lines 9 and 10 re-assigned here?

Reply:
Fixed